server/models/students.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+
+const { Schema } = mongoose;
+
+const PostSchema = new Schema(
+  { firstName: String, lastName: String, location: String, course: String },
+  {
+    timestamps: true,
+    versionKey: false,
+  }
+);
+
+module.exports = mongoose.model("Students", PostSchema);
```

Comment:
> Mongoose by default produces a collection name by passing the model name to the utils.toCollectionName method. This method pluralizes the name.
https://mongoosejs.com/docs/guide.html#collection

That means your model name should be "Student", in this case, and the collection in your DB `students`. I assume this is working because Mongoose checks for already pluralised schemas. You did it correctly in the `course` model! :)

Reply:
- and that's when I found about this ;) I'm changing the students too right now